### ISSUE TITLE
Repair advance-width artifacts caused by conversion from 2048 to 1000 unit design space

### ISSUE THREAD

**Issue by cmloegcmluin** [steinbergmedia/bravura#53]
We've noticed that many Sagittals have an advance-width of 187 when it should be 188. We've corrected that here. 

It's fine for v11 (from this issue: https://github.com/steinbergmedia/bravura/issues/52) to be used to obtain the *outlines* for the Olympian and Magrathean symbols (though this v12 includes that fix as well, if that's easier).

But it's probably easiest to simply update *widths* for *all Sagittals* per this attached v12:
[BravuraSagittalUpdate_v12.zip](https://github.com/steinbergmedia/bravura/files/7192890/BravuraSagittalUpdate_v12.zip)

Thanks again and let me know if you have any questions.

**Comment by cmloegcmluin** [comment]
It looks like the actual file, if I ever successfully attached it to the previous comment, is no longer there (perhaps editing the comment somehow resulted in its removal?)

In any case, here it is now: [BravuraSagittalUpdate_v12.zip](https://github.com/steinbergmedia/bravura/files/7763466/BravuraSagittalUpdate_v12.zip)

Sorry for any confusion.

**Comment by Jojo-Schmitz** [comment]
You had BravuraSagittalUpdate_v11.zip attached to #52, is that then obsolete?

**Comment by cmloegcmluin** [comment]
Yes, v11 is obsolete now. Thank you for getting that clarification.
